docs/af5-app-from-scratch/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,52 @@
+:navtitle: Introduction
+:reftext: Building an Axon Framework 5 Application from Scratch
+
+= Introduction
+
+Welcome to the Axon Framework 5 Getting Started Guide!
+This tutorial is based on the version `5.0.0-M1`.
+
+Building a new application from scratch can be daunting, especially when learning something new.
+This tutorial will guide you through the various stages of development,
+starting with creating your first project and explaining the capabilities of Axon Framework 5.
+
+The Axon Framework 5 introduces a lot of changes,
+so even if you are already familiar with Axon Framework 4, we still recommend you to go through this tutorial.
+
+== What's New in Axon Framework 5?
+Axon Framework 5 represents a significant evolution with major improvements to the framework's architecture:
+
+* Dynamic Consistency Boundary (DCB) - A revolutionary concept that allows more flexible event sourcing patterns overcoming traditional aggregates limitations. With this mind-shift you may organize your code more around features than entities. It plays nicely with Vertical Slice Architecture.
```

Comment:
I had never heard of this Vertical Slice Architecture before until a week ago. I bet readers of this don't either. 

Maybe link to https://www.baeldung.com/java-vertical-slice-architecture?

```suggestion
* Dynamic Consistency Boundary (DCB) - A revolutionary concept that allows more flexible event sourcing patterns overcoming traditional aggregates limitations. With this mind-shift you can organize your code  around features than entities with ease, called xref:https://www.baeldung.com/java-vertical-slice-architecture[Vertical Slice Architecture].
```

Reply:
The first/most-quoted article is this one:
https://www.jimmybogard.com/vertical-slice-architecture/

But I've checked briefly this from baeldung and looks even better, little similar to what I have done :) They also have events between, just do not use commands. I will apply this one!